lib/produce.csv

```diff
@@ -0,0 +1,535 @@
+erva-doce,green,BR,10
+espinafre,green,BR,10
+folha de uva,green,BR,10
+gobo,green,BR,10
+louro,green,BR,10
+manjericão,green,BR,10
+mostarda,green,BR,10
+nabo,green,BR,10
+rabanete,green,BR,10
+rúcula,green,BR,10
+abacate breda/margarida,fruit,BR,11
+abacaxi pérola,fruit,BR,11
+acerola,fruit,BR,11
+amêndoa,fruit,BR,11
+amora,fruit,BR,11
+avelã,fruit,BR,11
+banana prata,fruit,BR,11
+castanha nacional,fruit,BR,11
+coco verde,fruit,BR,11
+graviola,fruit,BR,11
+jaca,fruit,BR,11
+laranja pera,fruit,BR,11
+lima da pérsia,fruit,BR,11
+maçã nacional fuji,fruit,BR,11
+maçã estrangeira red delicious,fruit,BR,11
+mamão havaí,fruit,BR,11
+manga haden,fruit,BR,11
+manga palmer,fruit,BR,11
+manga tommy,fruit,BR,11
+maracujá doce,fruit,BR,11
+melão amarelo,fruit,BR,11
+nectarina nacional,fruit,BR,11
+nozes,fruit,BR,11
+sapoti,fruit,BR,11
+tangerina murcot,fruit,BR,11
+uva thompson,fruit,BR,11
+abobrinha italiana,vegetable,BR,11
+berinjela comum,vegetable,BR,11
+berinjela japonesa,vegetable,BR,11
+maxixe,vegetable,BR,11
+pepino japonês,vegetable,BR,11
+pepino caipira,vegetable,BR,11
+taquenoco,vegetable,BR,11
+tomate caqui,vegetable,BR,11
+cebola nacional,vegetable,BR,11
+cebola sp,vegetable,BR,11
+almeirão,green,BR,11
+aspargos,green,BR,11
+beterraba com folhas,green,BR,11
+brócolis,green,BR,11
+cenoura com folhas,green,BR,11
+couve de bruxelas,green,BR,11
+erva-doce,green,BR,11
+espinafre,green,BR,11
+folha de uva,green,BR,11
+gobo,green,BR,11
+hortelã,green,BR,11
+manjericão,green,BR,11
+mostarda,green,BR,11
+nabo,green,BR,11
+palmito,green,BR,11
+rabanete,green,BR,11
+abacaxi havaí,fruit,BR,12
+abacaxi pérola,fruit,BR,12
+acerola,fruit,BR,12
+ameixa nacional,fruit,BR,12
+amêndoa,fruit,BR,12
+amora,fruit,BR,12
+castanha nacional,fruit,BR,12
+cereja estrangeira,fruit,BR,12
+castanha estrangeira,fruit,BR,12
+coco verde,fruit,BR,12
+damasco estrangeiro,fruit,BR,12
+figo,fruit,BR,12
+framboesa,fruit,BR,12
+graviola,fruit,BR,12
+kiwi estrangeiro,fruit,BR,12
+laranja pera,fruit,BR,12
+lichia,fruit,BR,12
+limão taiti,fruit,BR,12
+maçã nacional fuji,fruit,BR,12
+maçã estrangeira red delicious,fruit,BR,12
+maçã estrangeira granny smith,fruit,BR,12
+manga haden,fruit,BR,12
+manga palmer,fruit,BR,12
+manga tommy,fruit,BR,12
+mangostão,fruit,BR,12
+maracujá doce,fruit,BR,12
+melancia,fruit,BR,12
+melão amarelo,fruit,BR,12
+nectarina nacional,fruit,BR,12
+nectarina estrangeira,fruit,BR,12
+nozes,fruit,BR,12
+pêssego nacional,fruit,BR,12
+physalis,fruit,BR,12
+romã,fruit,BR,12
+tâmara,fruit,BR,12
+uva itália,fruit,BR,12
+uva niagara,fruit,BR,12
+uva rubi,fruit,BR,12
+pimentão vermelho,vegetable,BR,12
+tomate caqui,vegetable,BR,12
+tomate,vegetable,BR,12
+tomate salada,vegetable,BR,12
+vagem,vegetable,BR,12
+erva-doce,green,BR,12
+folha de uva,green,BR,12
+gobo,green,BR,12
+hortelã,green,BR,12
+manjericão,green,BR,12
+mostarda,green,BR,12
+nabo,green,BR,12
+palmito,green,BR,12
+rabanete,green,BR,12
+rúcula,green,BR,12
+salsa,green,BR,12
+salsão,green,BR,12
```

Comment:
We need to add a new line here :) (more on that [here](https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log/5813359#5813359))